docs/deprecated_wiki/CentOS-install-guide.md

```diff
@@ -1,141 +0,0 @@
-For easy understanding of LORIS conventions and documentation, we recommend using the following account names: 
```

Comment:
This has been superseded by https://github.com/aces/Loris/tree/master/docs/wiki/00%20-%20SERVER%20INSTALL%20AND%20CONFIGURATION/01%20-%20LORIS%20Install/CentOS